packages/e2e/e2e-suite/sakuli.io/typed-check.ts

```diff
@@ -11,9 +11,11 @@
         await _wait( 5000, async () => (await driver.getAllWindowHandles()).length === 2);
         await driver.switchTo().window((await driver.getAllWindowHandles())[1]);
         await _highlight(_code("npm init"));
-        testCase.endOfStep("Find npm init code sample");
+        await driver.close();
+        await driver.switchTo().window((await driver.getAllWindowHandles())[0]);
```

Comment:
Why is this required now?
The driver should be closed by Sakuli. A manual driver close should not be required at any time.

Reply:
This is required because we now have two test cases opening a new tab.
Instead of using `CTRL/CMD+W` (we need the differentiation because Travis uses both Linux and macOS) to close a tab the webdriver already provides the [functionality](https://www.selenium.dev/documentation/en/webdriver/browser_manipulation/#closing-a-window-or-tab) with `driver.close()`. So with closing the tabs, it is no longer relevant in which order the test cases are run.

Reply:
Okay, good to know. Thx for clarification. :+1: